x/committee/spec/03_messages.md

```diff
@@ -17,22 +17,23 @@ type MsgSubmitProposal struct {
 
 ## State Modifications
```

Comment:
I would add the Committee interface, base committee, member committee, and token committee type definitions to 02_state.md